Agents/TripAgent/docker-compose-debug.yml

```diff
@@ -0,0 +1,7 @@
+services:
+  trip-agent:
+    image: trip-agent-debug:1.0
```

Comment:
I believe the standard convention for image tags would suggest "1.0.0", and I don't see any reason not to follow this here. Please fix. This applies to all the docker compose files.

Reply:
Will address in new repo